core-primitives/types/src/lib.rs

```diff
@@ -113,17 +114,21 @@ pub enum TrustedOperationStatus {
 #[derive(Encode, Decode, Clone, Debug, PartialEq)]
 pub enum WorkerRequest {
 	ChainStorage(Vec<u8>, Option<BlockHash>), // (storage_key, at_block)
+	/// for awareness: we call it unverified because there is no way how the enclave could verify the correctness of the information
+	LatestParentchainHeaderUnverified,
```

Comment:
Maybe we could add that this can be mitigated once we are running a light client.

Reply:
I disagree. the light client can't verify if the supplied block is indeed the latest one